create_references_for_ntjoin.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+# Split a fasta file into multiple files, one per sequence and create a reference config file for ntjoin
+# Usage: split_fasta.py <fasta file> <output directory> <config file>
+
+import sys
+
+input = sys.argv[1]
+output = sys.argv[2]
+config = sys.argv[3]
+ref_list = []
+with open(input, 'r') as f:
+    lines = f.readlines()
```

Comment:
It's good practice to read through files line-by-line, versus reading everything into memory, which isn't really necessary (and can inflate the memory usage). You can just see if the line is a header, and store it in a variable if so, ready to write the sequence once you get to the next line.
```suggestion
    for line in f:
        line = line.strip()
        // do the logic
```